phyui/plot/waveforms.py

```diff
@@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
     def __init__(self, **kwargs):
         super(WaveformView, self).__init__(**kwargs)
         self.visual = Waveforms()
+        self.resizable = Bool(defaultValue=True, sync=True)
```

Comment:
defaultValue comes from IPython.
but if you prefer value and it works I'am fine with it.

where is the IPython widget?